axelrod/match_generator.py

```diff
@@ -168,6 +168,32 @@ def estimated_size(self):
         size = self.__len__() * (1. / self.prob_end) * self.repetitions
         return size
 
+
+def test_graph_is_connected(edges, players):
+    """
```

Comment:
I think this might be better as a function returning a boolean - call it ```graph_is_connected``` and return True or False. The init methods then just need a line to raise an error if False.